plenum/common/metrics_collector.py

```diff
@@ -0,0 +1,67 @@
+import struct
+
+from abc import ABC, abstractmethod
+from enum import IntEnum
+from datetime import datetime, timezone
+from typing import Callable
+
+from storage.kv_store import KeyValueStorage
+
+
+class MetricType(IntEnum):
```

Comment:
It would be great to know how many node-to-node messages we have for 1 3PC Batch and 1 request